truss/templates/server/common/truss_server.py

```diff
@@ -267,9 +267,7 @@ def start(self):
             http="h11",
             host="0.0.0.0",
             port=self.http_port,
-            workers=self._config.get("runtime", {}).get(
-                "num_workers", DEFAULT_NUM_WORKERS
-            ),
+            workers=DEFAULT_NUM_WORKERS,
```

Comment:
somewhat nit: there are two notions of workers here:
* Uvicorn workers
* UvicornCustomServers

I think to minimize confusion here, we should govern these with different constants

Reply:
definitely, we should actually change the `config.runtime.num_workers` to something more apt, like `num_server_processes`?